model/src/main/java/com/gdgistanbul/model/Model.kt

```diff
@@ -4,38 +4,159 @@ import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
 
 @JsonClass(generateAdapter = true)
-data class EventContext(val host: Boolean)
+data class Event(
+    @Json(name = "created")
```

Comment:
@Json annotation is not needed if the json name and parameter name are same. so remove the ones that are not snake case.

Reply:
fixed